src/Panel/IncludePanel.php

```diff
@@ -137,16 +170,31 @@ protected function _isPluginFile($file)
     }
 
     /**
-     * Replace the path with APP, CORE or the plugin name
+     * Check if a path is from a Composer package
+     *
+     * @param string $file File to check
+     * @return bool
+     */
+    protected function _isVendorFile($file)
+    {
+        foreach ($this->_composerPaths as $package => $path) {
+            if (strstr($file, $path)) {
+                return $package;
```

Comment:
Is this of type bool?

Reply:
`strstr` returns string, but I just copy/pasted code from `_isPluginFile` .

Reply:
You return $package, but doc block states bool.